src/commcare_cloud/ansible/roles/ecryptfs/tasks/main.yml

```diff
@@ -25,7 +25,7 @@
 - name: Format block storage volume
   become: yes
   filesystem: fstype=ext4 dev="{{ datavol_device1 }}"
-  when: datavol_device is defined
+  when: datavol_device is defined and root_encryption_mode != 'aws'
```

Comment:
can you use the positive condition instead :`and root_encryption_mode == 'ecryptfs'`? That way seems clearer to me.

Also instead of adding the same condition to every task, could you wrap all of the tasks in the file in a single block that has that condition? Here's an example of somewhere where we use that pattern: https://github.com/dimagi/commcare-cloud/blob/982a3cf7e8283122edd004739bed280ef5a5bfba/src/commcare_cloud/ansible/roles/commcarehq/tasks/main.yml#L52-L100